tests/cxx/using_aliased_symbol-declare.h

```diff
@@ -0,0 +1,13 @@
+//===--- using_aliased_symbol-declare.h - test input file for iwyu --------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Just realized this is a header, which is why there's no test description; there's no test.